session/bootstrap.go

```diff
@@ -339,6 +339,22 @@ const (
 		WITH_GRANT_OPTION enum('N','Y') NOT NULL DEFAULT 'N',
 		PRIMARY KEY (USER,HOST,PRIV)
 	  );`
+
+	// CreatePlacementPolicyTable store the placement policys.
+	CreatePlacementPolicyTable = `CREATE TABLE IF NOT EXISTS mysql.placement_policy (
+		NAME varchar(255) NOT NULL,
```

Comment:
Tables/views are limited to 64. I think we should chose similar for consistency:
```suggestion
		NAME varchar(64) NOT NULL,
```

Reply:
Yes, sure